rskj-core/src/main/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderHardcoaded.java

```diff
@@ -0,0 +1,25 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.script.Script;
+import org.bouncycastle.util.encoders.Hex;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class NonStandardErpRedeemScriptBuilderHardcoaded implements ErpRedeemScriptBuilder {
```

Comment:
```suggestion
public class NonStandardErpRedeemScriptBuilderHardcoded implements ErpRedeemScriptBuilder {
```
small typo, remember to also change the file name